signinapp/admin/users.py

```diff
@@ -3,19 +3,27 @@
 from flask_wtf import FlaskForm
 from werkzeug.security import generate_password_hash
 from wtforms import (BooleanField, PasswordField, SelectField, StringField,
-                     SubmitField)
+                     SubmitField, TelField, EmailField)
 from wtforms.validators import DataRequired, EqualTo
 
-from ..model import Role, Subteam, User, db
+from ..model import Role, Subteam, User, db, ShirtSizes
 from ..util import admin_required
 from .util import admin
 
 
 class UserForm(FlaskForm):
     name = StringField(validators=[DataRequired()])
+    preferred_name = StringField("Preferred Name")
     password = PasswordField()
     role = SelectField()
     subteam = SelectField()
+
+    phone_number = TelField("Phone Number")
+    email = EmailField("Email Address")
+    address = StringField("Street Address")
```

Comment:
*Technically* other teams (or even us in weird cases) might want to have the address include the town, but since this is just a string it's fine as is.

Reply:
We could change this to just Address? The street is hopefully all we care about, and I don't want/need everyone spending a bunch of time entering their full address. I'll leave as-is for now, we can change the form in the future.